api/envoy/service/discovery/v2/csds.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package envoy.service.discovery.v2;
+
+option java_package = "io.envoyproxy.envoy.service.discovery.v2";
+option java_outer_classname = "CsdsProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+
```

Comment:
This should live in `envoy.service.status.v2` as per the new package layout https://github.com/envoyproxy/envoy/pull/9555

Reply:
Done.